Dizzy/Entities/ExternalNavigationProvider.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ExternalNavigationProvider.swift
+//  Dizzy
+//
+//  Created by stas berkman on 04/07/2019.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import UIKit
+
+class ExternalNavigationProvider {
+
+    func openWaze(location : Location) {
+        if let url = URL(string: "waze://"), UIApplication.shared.canOpenURL(url) {
+            let urlStr: String = "waze://?ll=\(location.latitude),\(location.longitude)&navigate=yes"
+            if let url = URL(string: urlStr) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
+            }
+        } else {
+            // Waze is not installed. Launch AppStore to install Waze app
```

Comment:
instead of writing mark, please move this code in to function called `askToInstallWaze`